leetcode/math.md

```diff
@@ -0,0 +1,24 @@
+# Math
```

Comment:
Your pull request: LEETCODE-1022 Added base-7


Your commit: LEETCODE-1022 Added base-7

Your commit: LEETCODE 1022 Refactored base-7
The prefix of your comment must be from ['LEETCODE', 'GENERATOR', 'TRIANGLE', 'HEXNUMBER']
Number of your group must be from ['1021', '1022']
Your commit: LEETCODE-1022 Refactored base-7
Action of your comment must be from ['Added', 'Deleted']
Your commit: Merge remote-tracking branch 'origin/base-7' into base-7

# Conflicts:
#	leetcode/math.md
The prefix of your comment must be from ['LEETCODE', 'GENERATOR', 'TRIANGLE', 'HEXNUMBER']
Number of your group must be from ['1021', '1022']
Action of your comment must be from ['Added', 'Deleted']
Your commit: LEETCODE-1022 Refactored base-7
Action of your comment must be from ['Added', 'Deleted']
Your commit: LEETCODE-1022 Refactored base-7
Action of your comment must be from ['Added', 'Deleted']